decidim-elections/config/locales/en.yml

```
@@ -137,6 +139,28 @@ en:
           update:
             invalid: There was a problem updating this question
             success: Question successfully updated
+        trustee_participatory_space:
+          actions:
+            disable: disable
```

The first letter should be capitalized 😀

```suggestion
            disable: Disable
```